package-lock.json

```diff
@@ -2708,7 +2708,7 @@
     "node_modules/immediate": {
       "version": "3.0.6",
       "resolved": "https://registry.npmjs.org/immediate/-/immediate-3.0.6.tgz",
-      "integrity": "sha1-nbHb0Pr43m++D13V5Wu2BigN5ps=",
+      "integrity": "sha512-XXOFtyqDjNDAQxVfYxuF7g9Il/IbWmmlQg2MYKOH8ExIT1qg6xc4zyS3HaEEATgs1btfzxq15ciUiY7gjSXRGQ==",
```

Comment:
I don't understand what dependencies you updated.  Did you make any updates to dependency versions in package.json?  Or just run a fresh "npm install" with the existing dep versions?

Reply:
I just ran `npm install` and followed the instruction to fix the Invalid URL error by selenium-webdriver. It automatically updated some information including this.